content/100-getting-started/02-setup-prisma/200-add-to-existing-project/110-relational-databases/150-introspection.mdx

```diff
@@ -390,6 +879,32 @@ Learn more about this on the [Configuring your Prisma Client API](/concepts/comp
 
 </SwitchTech>
 
+<SwitchTech technologies={['node', 'planetscale']}>
+
+<NavigationLinksContainer>
+
+<ButtonLink
+  color="dark"
+  type="primary"
+  href="./connect-your-database-node-planetscale"
+  arrowRight
```

Comment:
Should be arrowLeft probably?
![image](https://user-images.githubusercontent.com/183673/149135712-5aeefe23-f8b5-40d5-97e6-9072c3fe8f20.png)

Reply:
Done